mango-cli/src/main/webapp/js/utilities.js

```diff
@@ -26,6 +26,16 @@ function toggleAlignments(sample, selector) {
         $(selector).slideToggle( "fast" );
 }
 
+// Toggles alignment record contained in the selector for a given sample
+function toggleVariants() {
```

Comment:
did you add this? I think I did. It's a useless function should get rid of it, as with the setGlobalMapQ method below. If not, I'll remove it with my changes.